packages/mco/components/disaster-recovery/create-dr-policy/create-dr-policy.tsx

```diff
@@ -82,11 +83,10 @@ export const CreateDRPolicy: React.FC<ReRouteResourceProps> = ({
     specName: ODFMCO_OPERATOR,
     cluster: HUB_CLUSTER_NAME,
   });
-  const odfMCOVersion = React.useMemo(() => {
-    const version = csv?.spec?.version;
-    // converting z-stream release version to main release version
-    return version ? version.substring(0, version.lastIndexOf('.')) + '.0' : '';
-  }, [csv]);
+  const odfMCOVersion = React.useMemo(
+    () => getMajorVersion(csv?.spec?.version),
+    [csv]
+  );
```

Comment:
```suggestion
  const odfMCOVersion = getMajorVersion(csv?.spec?.version);
```
useMemo is for expensive operations. No need here...